test/api/utils.js

```diff
@@ -33,7 +33,7 @@ describe('cheerio', function () {
       var $ = cheerio.load('<div>a div</div><span>a span</span>');
       var $collection = $('div').add($.root()).add('span');
       var expected =
-        '<span>a span</span><html><head></head><body><div>a div</div><span>a span</span></body></html><div>a div</div>';
```

Comment:
These elements used to be disjoint, because `uniqueSort` did not resolve the `root` reference. Now, `uniqueSort` will actually put the document first & the span last (as it should).